Bluejay/Bluejay/Error+Extensions.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  Error+Extensions.swift
```

Comment:
I don't see any compelling reason for this stuff to be in a separate file. I think it should just be moved into Error.swift

Reply:
Done